test/e2e/storage/persistent_volumes-disruptive.go

```diff
@@ -159,15 +156,17 @@ var _ = framework.KubeDescribe("PersistentVolumes [Volume][Disruptive][Flaky]",
 func testKubeletRestartsAndRestoresMount(c clientset.Interface, f *framework.Framework, clientPod *v1.Pod, pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolume) {
 	By("Writing to the volume.")
 	file := "/mnt/_SUCCESS"
-	_, err := podExec(clientPod, fmt.Sprintf("touch %s", file))
+	out, err := podExec(clientPod, fmt.Sprintf("touch %s", file))
+	framework.Logf(out)
 	Expect(err).NotTo(HaveOccurred())
 
 	By("Restarting kubelet")
 	kubeletCommand(kRestart, c, clientPod)
 
 	By("Testing that written file is accessible.")
-	_, err = podExec(clientPod, fmt.Sprintf("cat %s", file))
+	out, err = podExec(clientPod, fmt.Sprintf("cat %s", file))
 	Expect(err).NotTo(HaveOccurred())
+	framework.Logf(out)
```

Comment:
Here the Expect is before the Logf, but L160 the Logf occurs before the Expect. Should be consistent.